### PR TITLE
Add ADR documenting change in Distributed Workloads deployment strategy

### PR DESCRIPTION
**Pull request by anishasthana**
cc @LaVLaS @etirelli @danielezonca @jwforres @VaishnaviHire @astefanutti @dimakis @sutaakar

### CODE REVIEW

**Review by etirelli — APPROVED** [review]

File: distributed-workloads/ODH-ADR-DW-0001-determine-codeflare-deployment-strategy.md [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+      1. Having a fork allows us to have an ODH-controlled repository for new image builds and manifest references, allowing us to better version the CodeFlare stack in ODH.
+2. CodeFlare CRDs will initially be included alongside the CFO manifests
+   1. They will eventually be moved into the ODH Operator bundle.
+   2. An implication of this is that ODH and community olm CFO can not be installed on the same cluster.
+3. This fork will be synced with upstream every time there is an upstream release
+4. As we also have a fork of KubeRay in the ODH organization, we can follow a similar process for KubeRay. Given this, we can probably delete all manifests from the distributed workloads repository.
+
+This would simplify the user experience greatly as users simply need to enable CodeFlare in the DSC, with ODH taking care of the rest. The above flow also brings CodeFlare closer in line with other ODH components. From a testing perspective, all of our existing test cases will still be useful. The only ones that won’t carry over without updates are the existing olm upgrade tests in the upstream CFO repository.
```
Comment (VaishnaviHire) [inline distributed-workloads/ODH-ADR-DW-0001-determine-codeflare-deployment-strategy.md:49]: For an existing deployment of codeflare operator, will ODH operator prevent further deployment and update status asking user to `uninstall` the operator?

Comment (anishasthana) [reply distributed-workloads/ODH-ADR-DW-0001-determine-codeflare-deployment-strategy.md:49]: I think that's what the behavior should end up being.

**Review by VaishnaviHire — APPROVED** [review]
/lgtm

**Comment by anishasthana** [comment]
@LaVLaS @etirelli @gregsheremeta could one of you merge this PR please?

**Review by danielezonca — APPROVED** [review]

**Pull request merged**